DataFormats/Detectors/FIT/FDD/include/DataFormatsFDD/RecPoint.h

```diff
@@ -15,23 +15,77 @@
 #define ALICEO2_FDD_RECPOINT_H
 
 #include "CommonDataFormat/InteractionRecord.h"
+#include "CommonDataFormat/InteractionRecord.h"
+#include "CommonDataFormat/TimeStamp.h"
+#include "DataFormatsFDD/ChannelData.h"
+#include "CommonDataFormat/RangeReference.h"
+#include "DataFormatsFDD/Digit.h"
 
 namespace o2
 {
 namespace fdd
 {
+struct ChannelDataFloat {
+
+  int mPMNumber = -1;         // channel Id
+  double mTime = -20000;      // time in ps, 0 at the LHC clk center
+  double mChargeADC = -20000; // charge [channels]
+  int adcId = -1;             // QTC chain
```

Comment:
For the memory alignment reasons better group data members of the same size together, i.e. 1st 32bit ints, then 64bit doubles.

Reply:
Done